Superposition/ReferenceImplementation.qs

```diff
@@ -231,16 +231,16 @@ namespace Quantum.Kata.Superposition {
     // Example: for bit string = [true, false] the qubit state required is (|00⟩ + |10⟩) / sqrt(2).
     operation ZeroAndBitstringSuperposition_Reference (qs : Qubit[], bits : Bool[]) : Unit is Adj {
 
-        EqualityFactI(Length(bits), Length(qs), "Arrays should have the same length");
-        EqualityFactB(bits[0], true, "First bit of the input bit string should be set to true");
+        Fact(Length(bits)==Length(qs), "Arrays should have the same length");
```

Comment:
I'm pretty sure we had this discussion some time ago and you said Fact is preferable to EqualityFactX because it's more readable? :-) 

This particular message is more for me than for the end user (they're not going to get it if I wrote the tests correctly), so reporting the exact lengths is not critical.

Reply:
I think that was specific to using `EqualityFactB` to compare to a literal true, sorry for not being more clear; that's on me. More generally, it'd be good if we could explain facts a bit better in docs so that that's a resource available for this kind of thing.